Microsoft.Toolkit.Uwp/Helpers/ConnectionHelper.cs

```diff
@@ -106,5 +109,54 @@ public static ConnectionType ConnectionType
                 return connectionType;
             }
         }
+
+    /// <summary>
+    /// Gets connection ssid for the current Wifi Connection.
+    /// </summary>
+    /// <returns> string value of current ssid/></returns>
+    ///
+    public static async Task<string> GetNetworkSsidAsync()
```

Comment:
yes I agree with @hermitdave. I want to be sure that there is an ask for this PR first. So as soon as the Uservoice will be open, I'll make sure to create an entry for this PR and absed on feedback we will see what to do